src/man/sssd.conf.5.xml

```diff
@@ -560,6 +560,15 @@
                                 in a random order for each parent domain.
                             </para>
                             <para>
+                                Note that when this option is set the output
+                                format of all commands is always
+                                fully-qualified, even when using short names.
+                                In case the administrator wants the output not
+                                fully-qualified, the full_name_format option
+                                can be used as shown below:
+                                <quote>full_name_format=%1$s</quote>
+                            </para>
+                            <para>
```

Comment:
IIUC it is the same case as with `default_domain_suffix` (in 2nd paragraph)
```
       default_domain_suffix (string)
           This string will be used as a default domain name for all names
           without a domain name component. The main use case is environments
           where the primary domain is intended for managing host policies and
           all users are located in a trusted domain. The option allows those
           users to log in just with their user name without giving a domain
           name as well.

           Please note that if this option is set all users from the primary
           domain have to use their fully qualified name, e.g.
           user@domain.name, to log in. Setting this option changes default of
           use_fully_qualified_names to True. It is not allowed to use this
           option together with use_fully_qualified_names set to False.

           Default: not set
```
or is it a different case?

Reply:
And answering to my own question is that it is a different case :-) It was enough to check patches in related ticket
https://pagure.io/SSSD/sssd/issue/3403.